### PR TITLE
Update website copy to match updated README

### PR DESCRIPTION
**Pull request by jbw976**
This PR updates the website copy to match the recently updated README from https://github.com/crossplane/crossplane/pull/2198/.

The overall structure and layout of the site is left the same, but the copy has been updated with new messaging.

This has been tested in a local browser using `make run`, and a preview site with the changes can be browsed at https://jbw976.github.io/crossplane.github.io/.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -60,8 +63,8 @@ <h2>Provision and manage cloud infrastructure and services using kubectl</h2>
           alt="Stacks" id="stacksImg" />
         <h2>Created to power a more open cloud</h2>
         <p>
-          We built Crossplane to help organizations build their clouds like the cloud vendors build theirs—with a control plane. Crossplane is a CNCF sandbox project which extends the Kubernetes 
-          API to manage and compose infrastructure. Operators can encapsulate policies, permissions, and other guardrails behind a custom API line generated by Crossplane while application developers 
+          We built Crossplane to help organizations build their clouds like the cloud vendors build theirs—with a control plane. Crossplane is a CNCF sandbox project which extends the Kubernetes
```
Comment (hasheddan) [inline index.html:66]: nitpick:
```suggestion
          We built Crossplane to help organizations build their clouds like the cloud vendors build theirs — with a control plane. Crossplane is a CNCF sandbox project which extends the Kubernetes
```

**Review by hasheddan — APPROVED** [review]
One minor nitpick, but LGTM!

File: index.html (hunk 1 above)
Comment (connorchan) [reply index.html:66]: Most style guides push for not using spaces around em dashes. I'm not that picky, though 🤷

**Review by connorchan — APPROVED** [review]

File: index.html [hunk 2]
```diff
@@ -106,13 +109,12 @@ <h2>Publish simplified infrastructure abstractions for your applications</h2>
     </div>
     <div class="grid">
       <div class="col-6_md-12">
-        <h2>Deploy applications using a team centric approach with OAM</h2>
+        <h2>Universal Cloud API</h2>
```
Comment (grantgumina) [inline index.html:112]: Suggestion: "The Universal Cloud API"

File: index.html [hunk 3]
```diff
@@ -106,13 +109,12 @@ <h2>Publish simplified infrastructure abstractions for your applications</h2>
...
         <p>
-          Crossplane implements the Open Application Model to help unify application and infrastructure management around team-centric processes. With Crossplane and OAM, 
-          application and infrastructure config can co-exist and be deployed using the same tooling.
-        </p>
-        <p>
-          <a href="https://oam.dev">Learn More About OAM</a>
+          Crossplane provides a consistent API across a diverse set of vendors, resources, and
+          abstractions. The Crossplane Resource Model (XRM) extends the Kubernetes Resource Model
+          (KRM) in an opinionated way, resulting in a universal experience for managing resources,
```
Comment (grantgumina) [inline index.html:116]: "uniform experience"

File: index.html [hunk 4]
```diff
@@ -106,13 +109,12 @@ <h2>Publish simplified infrastructure abstractions for your applications</h2>
...
+          no matter what provider or level of abstraction they are a part of.
```
Comment (grantgumina) [inline index.html:117]: Suggestion: "no matter what provider or vendor built them"

The "level of abstraction" is hard for me to grok.

**Comment by jbw976** [comment]
Thanks @grantgumina, incorporated all your suggestions!

**Pull request merged**